src/Designer/backend/tests/Designer.Tests/Helpers/Extensions/StringExtenionsTests.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Altinn.Studio.Designer.Helpers.Extensions;
+using Xunit;
+
+namespace Designer.Tests.Helpers.Extensions;
+
+public class StringExtensionsTests
+{
+    [Fact]
+    public void WithoutPrefix_WhenStringStartsWithGivenPrefix_ReturnsWithoutPrefix()
+    {
+        string original = "Lorem ipsum";
+        string result = original.WithoutPrefix("Lorem");
+        Assert.Equal(" ipsum", result);
+    }
+
+    [Fact]
+    public void WithoutPrefix_WhenStringDoesNotStartWithGivenPrefix_ReturnsOriginalString()
+    {
+        string original = "Lorem ipsum";
+        string result = original.WithoutPrefix("ipsum");
+        Assert.Equal(original, result);
```

Comment:
Just to lock down the test
```suggestion
        Assert.Equal("Lorem ipsum", result);
```